docs/storage.md

```diff
@@ -88,9 +88,7 @@ pvc-9a7fae2d-eb03-42c3-aaa9-1a807d5df12f   5Gi        RWO            Delete
 
 ## CSI
 
-K0s also supports any other Kubernetes storage solutions with CSI.
-
-k0s supports a wide range of different storage options. There are no "selected" storage in k0s. Instead, all Kubernetes storage solutions are supported and users can easily select the storage that fits best for their needs.
+k0s supports a wide range of different storage options by utilizing Container Storage Interface (CSI). There are no "selected" storage in k0s. Instead, all Kubernetes storage solutions are supported and users can easily select the storage that fits best for their needs.
```

Comment:
I think this sentence doesn't apply anymore, so some refactoring needed:
"There are no "selected" storage in k0s."